hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSender.java

```diff
@@ -214,6 +217,12 @@ public void updateSubscription(Subscription newSubscription) {
       this.messageSender.stop();
       this.messageSender = messageSender(newSubscription);
     }
+    if (subscriptionMetricsConfigChanged) {
+      this.successHandlers.stream()
+          .filter(SubscriptionChangeListener.class::isInstance)
```

Comment:
why do we need this? actually there is only one success handler so it does not change anything 🤔